scripts/zones/Chocobo_Circuit/npcs/Adrian.lua

```diff
@@ -1,7 +1,8 @@
 -----------------------------------
 -- Area: Chocobo_Circuit
---  NPC: Adrian
--- Standard Info NPC
+-- NPC: Adrian
+-- Abyssea Support NPC
+-- POS: -330.5837 0.0000 -468.5058
```

Comment:
The rest of the codebase has the pos comment set so that you can copy/paste it to go straight to the NPC

Example:
```lua
-- !pos 230.413 32.278 280.677 15
```

Also (not that it matters much), they're all 3 decimal places or less

Reply:
I added the zone value based on the !pos command "func: pos <x> <y> <z> <optional zone> <optional target>" I will begin updating the others to match the three pos style